src/webrtc/gcc.md

```diff
@@ -0,0 +1,159 @@
+# WebRTC Congestion Control
+
+<!-- 1. Congestion Control
+2. TWCC vs GCC
+3. GCC
+4. GCC & Simulcast
+5. Connection Probing
+6. Strategies for probing the connection -->
+
+
+## Congestion Control
+
+The network has its capacity. When a sender sends more data than network devices are able to process,
+packets will be delayed and finally dropped. Such situation is called the network congestion.
+
+Congestion control goal is to prevent from congesting the network by controling the sender rate.
+Every time, the sender notices that the packet transmission time increases or packets are getting
+lost it should lower its rate.
+
+<!-- In the case of multimedia, lowering sender rate is challenging as the audio and video 
+have specified bitrate and sometimes even lowering the video quality might not  -->
+
+## TWCC vs GCC
+
+TWCC (Transport-wide Congestion Control) defines new RTP header extension and RTCP message used to 
+feedback the arrival times of RTP packets to the sender.
+Each RTP packet is assigned transport-wide (unique across all RTP streams sent at the same socket) 
+sequence number by the sender before transmission.
+The receiver records arrival timestamps of packets and send them back in TWCC Feedback message.
+Then, the sender is able to determine network delay and packet loss, and react to them by lowering its rate.
```

Comment:
I think the first part of the sentence doesn't make it clear that we're talking about significant network delay and significant packet loss.
I also have a problem with `its rate`. You didn't introduce the sender rate yet. Let's do `amount of data it sends`, shall we?
Maybe we should rewrite it as...
```suggestion
Then the sender is able to determine network delay and packet loss and react to them by adjusting the amount of data it sends, increasing bitrate if network conditions are good or decreasing it if they aren't.
```

Also, a pro tip, use Grammarly or something. In English, a comma is used rather sparsely and it will highlight the misuse of commas, as well as other useful things.

Reply:
>  In English, a comma is used rather sparsely and it will highlight the misuse of commas, as well as other useful things.

I will, this PR wasn't meant for CR yet :D

Reply:
> I think the first part of the sentence doesn't make it clear that we're talking about significant network delay and significant packet loss.

That's only an introduction/general description. The details (e.g. packet loss threshold triggering the reaction) can be provided later

> I also have a problem with its rate. You didn't introduce the sender rate yet. Let's do amount of data it sends, shall we?

You mean that it's unclear what `its` refers to? The `sender rate` was used in the `Congestion Control` section for the first time